website/build.gradle

```diff
@@ -80,10 +80,10 @@ task createDockerContainer(type: Exec) {
     } else {
       // Otherwise: run as current user, so as to not require sudo to clean up
       // build/ directory.
-      extraOptions += " -u \$(id -u):\$(id -g)"
+      extraOptions += " -u 0:0"
```

Comment:
Is this safe? Why are we making this change?

Reply:
I changed to a more restricted user, it is required to have permissions on opt/ dir, without this change a permissions error is shown as in https://github.com/benWize/beam/actions/runs/3251224733/jobs/5335936519